test/parallel.jl

```diff
@@ -1,8 +1,12 @@
+
+cd(cwd)
```

Comment:
`cwd` is not declared when running this file individually.

Reply:
well supposedly you always have to run `io.jl` before running this one, or else you have no files, thats why I put it. but yeah I see what you mean, I removed it ;)